README.md

```diff
@@ -67,6 +67,36 @@ if doing this for the first time. Here's a quick summary:
     </div>
     ```
 
+    Playlist:
+
+    ```html
+    <div class="jp-gui jp-interface">
+        <div class="jp-controls">
+            <a class="jp-previous"><i class="fa fa-backward"></i></a>
```

Comment:
This is almost 100% identical to the html sample above, but it's missing the container wrapper and has the addition of `jp-previous` and `jp-next`. I think it would be better to simply make a note about adding these two controls rather than duplicating the entire sample.